src/io/flutter/jxbrowser/JxBrowserManager.java

```diff
@@ -58,8 +56,35 @@ public static JxBrowserManager getInstance() {
     return manager;
   }
 
-  public boolean isInstalled() {
-    return status.get().equals(JxBrowserStatus.INSTALLED);
+  public JxBrowserStatus getStatus() {
+    return status.get();
+  }
+
+  public JxBrowserStatus waitForInstallation(int seconds) throws TimeoutException {
```

Comment:
nit: comment that you must call setup as well as calling this message or it will time out.